nodebuilder/daser/config.go

```diff
@@ -0,0 +1,76 @@
+package daser
+
+import (
+	"errors"
+	"time"
+)
+
+var (
+	ErrNoSamplingRange = errors.New("nodebuilder/daser: misconfiguration error, sampling range cannot be 0")
+	ErrNoConcLimit     = errors.New("nodebuikder/daser: misconfiguration error, concurrency limit cannot be 0")
```

Comment:
Typo in the `builder`. But anyway, in some other PR we agreed to call our module pkg as `modp2p`, `modheader`, and `moddaser` instead of `nodebuilder/daser`.